src/Illuminate/Foundation/helpers.php

```diff
@@ -253,7 +253,7 @@ function broadcast_if($boolean, $event = null)
     function broadcast_unless($boolean, $event = null)
     {
         if (! $boolean) {
-            return app(BroadcastFactory::class)->event($event);
+            return broadcast($event);
         } else {
             return new FakePendingBroadcast;
         }
```

Comment:
Wouldn't delegation to `broadcast_if()` be consequential here?
```suggestion
        return broadcast_if(!$boolean, $event);
```

Reply:
I also think that, but I will leave that decision to @taylorotwell